docs/auth-directive.md

```diff
@@ -188,3 +188,74 @@ app.register(mercuriusAuth, {
 
 app.listen({ port: 3000 })
 ```
+
+## Mutliple Auth Roles Directive
+
+The auth directive can be modified to allow multiple roles to be passed in.
+
+```js
+'use strict'
+
+const Fastify = require('fastify')
+const mercurius = require('mercurius')
+const mercuriusAuth = require('..')
```

Comment:
Minor nit to improve the docs

```suggestion
const mercuriusAuth = require('mercurius-auth')
```

Reply:
I've made the change you suggested and also fixed the lint errors